spring-cloud-data-rest/src/main/java/org/springframework/cloud/data/AdminApplication.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package org.springframework.cloud.data.rest;
```

Comment:
Hmm, not sure what you're saying. The "ui" part is not a REST-API. It's a regular `.html` app, that is _consuming_ the API.
I understand what you're saying wrt namespacing. IMO, there are two solutions:
* go back to having the `ui` serving part inside `.rest` and live with it
* create an extra project, which is the real "admin", made of 2 parts: the REST endpoints and the ui